homeassistant/util/color.py

```diff
@@ -210,6 +217,11 @@ def color_RGB_to_xy(iR: int, iG: int, iB: int) -> Tuple[float, float, int]:
     return round(x, 3), round(y, 3), brightness
 
 
+def color_xy_to_RGB(vX: float, vY: float) -> Tuple[int, int, int]:
```

Comment:
As this is now only used in `color_xy_to_hs()`, would you mind inlining it to avoid the potential confusion?

Reply:
I'd rather keep it separate, so it's available to be used in the future if platforms need it, and more importantly, we've already got separate tests for it.